test/AbstractFactory/ReflectionBasedAbstractFactoryTest.php

```diff
@@ -171,7 +174,7 @@ public function testFactoryCanSupplyAMixOfParameterTypes()
         self::assertInstanceOf(TestAsset\ClassWithMixedConstructorParameters::class, $instance);
 
         self::assertEquals($config, $instance->config);
-        self::assertEquals([], $instance->options);
+        self::assertEquals(null, $instance->options);
```

Comment:
Does this change mean that there was an error before?

Reply:
Yes, there was an error in the tests when I ran them locally. I thought I saw a comment something about `$options` always being treated as `null`, but now I can't find it. I'll revert this change and then you can see the error that shows up in Travis.

Reply:
@froschdesign Yep, in the test the `options` instance property is set to `null`, not to an empty array, see:
https://github.com/laminas/laminas-servicemanager/blob/3.5.x/test/AbstractFactory/TestAsset/ClassWithMixedConstructorParameters.php#L24

So either the assertion or the test class constructor needs changing I guess.